bigquery/bigquery_create_reservation/test.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Google LLC
```

Comment:
Adding this file will no longer test the bq reservation resource. Is this intended?